test_rigs/unit-tests/configs/cisco_hsrp

```diff
@@ -35,12 +35,14 @@ interface TenGigE0/0/2/2
  hsrp delay minimum 0 reload 0
  no hsrp use-bia
  hsrp 123
+  authentication cisco
   authentication text blah
-  name bloop
+  ip 1.2.3.4
   mac-address 0123.4567.89AB
+  name bloop
+  no preempt
```

Comment:
Please do not "sort" configs while adding new features -- it causes needless churn in the refs that is hard to review

Reply:
Acknowledged.